src/main/resources/schemas/nhinc/common/configAdmin.xsd

```diff
@@ -36,4 +36,65 @@
     </xsd:complexType>
     <xsd:element name="ImportCertificateResponseMessage" type="tns:ImportCertificateResponseMessageType"/>
     
+    
+    <xsd:complexType name="ListTrustStoresRequestMessageType" >
+        <xsd:sequence>
+            <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
```

Comment:
For the list trust stores request can you add another element:
<xsd:element name="refreshCertCache" minOccurs="1" maxOccurs="1" default="false" />